app/middleware/login.js

```diff
@@ -52,36 +68,27 @@ export default (store: Store) =>
           AbortionSystem.abort(ABORTION_ERROR);
           await store.dispatch(setLoginStatus(LOGINSTATUS.ERROR));
         }
-        await sleep(2000);
-      } else {
-        await store.dispatch(logout());
-      }
-    } else if (action.type === LOGOUT) {
-      await store.dispatch(setBotIsActing(false));
-      await store.dispatch(
-        electronRouting('puppet', 'https://www.immobilienscout24.de/')
-      );
-      await sleep(4000);
-      const { webContents } = electronObjects.views.puppet;
-      const electronUtils = new ElectronUtilsRedux(webContents, store.dispatch);
-      const { abortableAction: abortablePerformLogout, abort } =
-        abortable(performLogout);
-      AbortionSystem.registerAbort(abort);
-      try {
-        await timeout(
-          abortablePerformLogout(
-            store.dispatch,
-            electronUtils,
-            store.getState().configuration
-          ),
-          300000
-        );
-      } catch (error) {
-        // eslint-disable-next-line no-console
-        console.error(`Error during logout: ${error}`);
-        AbortionSystem.abort(ABORTION_ERROR);
-        await store.dispatch(setLoginStatus(LOGINSTATUS.ERROR));
+      } else if (useAccount == USEACCOUNT.MANUELL) {
+        const { abortableAction: abortablePerformManualLogin, abort } =
+          abortable(performManualLogin);
+        AbortionSystem.registerAbort(abort);
+        try {
+          await timeout(
+            abortablePerformManualLogin(
+              store.dispatch,
+              electronUtils,
+              store.getState().configuration
+            ),
+            300000
+          );
+        } catch (error) {
+          // eslint-disable-next-line no-console
+          console.error(`Error during login: ${error}`);
+          AbortionSystem.abort(ABORTION_ERROR);
+          await store.dispatch(setLoginStatus(LOGINSTATUS.ERROR));
```

Comment:
Just a question because I can't remember: do you know if and where this error is communicated to the user?

Reply:
I didn't see any... would have to double check